### PR TITLE
Add PayPal information into the 'Edit Profile' page

### PR DESCRIPTION
**Pull request by rupertbates**
## What does this change?
Adds code to the membership tab of the edit profile page to handle users who have PayPal as a payment method. Trello card [here](https://trello.com/c/eOtuebKl/276-update-members-data-api-to-cope-with-paypal-users)

## What is the value of this and can you measure success?
It supports the work to enable PayPal as a payment method within Membership, with the ultimate aim of improving conversion rates, particularly on mobile.

## Does this affect other platforms - Amp, Apps, etc?
Web only

## Screenshots
screen shot 2017-01-18 at 12 02 09
screen shot 2017-01-18 at 12 02 23

## Tested in CODE?
Can't test this on code because it requires https

### CODE REVIEW

File: identity/app/views/fragments/profile/payPal.scala.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@import views.html.fragments.selectDates
+@()
```
Comment (Ap0c) [inline identity/app/views/fragments/profile/payPal.scala.html]: ?

File: identity/app/views/fragments/profile/payPal.scala.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <div class="fieldset__heading">
+            <h2 class="form__heading">
+                Payment method
+            </h2>
+        </div>
+
+        <div class="fieldset__fields">
+            <div class="form-field">
+                <span class="paypal-message">
+                You paying with PayPal.
```
Comment (Ap0c) [inline identity/app/views/fragments/profile/payPal.scala.html]: You're?

**Review by Ap0c — APPROVED** [review]
Runs locally, looks good! ⛵️

**Review by paulbrown1982 — APPROVED** [review]
👍 LGTM

**Pull request merged**